### PR TITLE
Fixes #16903: Document /var/rudder/policy-generation-info

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/16903

### CODE REVIEW

File: src/reference/modules/usage/pages/advanced_configuration_management.adoc [hunk 1]
```diff
@@ -36,6 +36,74 @@ the regeneration of all policies. It will clear all internal caches, and force a
...
+more specifics info like: where are info and debug log? 
+
+==== /var/rudder/policy-generation-info
+
+Directory `/var/rudder/policy-generation-info` contains information about the last(s) policy generation
+in the following files:
+
+- `last-failure-message`: when a generation fails, failure message is store here. It is helpful for
+  debugging purpose (for example when message is very long), or to know what happened in case of a 
+  transcient errors.
```
Comment (ncharles) [inline src/reference/modules/usage/pages/advanced_configuration_management.adoc]: transient errors

File: src/reference/modules/usage/pages/advanced_configuration_management.adoc [hunk 2]
```diff
@@ -36,6 +36,74 @@ the regeneration of all policies. It will clear all internal caches, and force a
...
 
+=== Policy generation info log
+
+Policy generation is a complex process and it is central to Rudder use. You will often want to know more
+about a policy generation result: what was the reason for a failure? What nodes where upated? Or even 
+more specifics info like: where are info and debug log? 
+
+==== /var/rudder/policy-generation-info
+
+Directory `/var/rudder/policy-generation-info` contains information about the last(s) policy generation
```
Comment (ncharles) [inline src/reference/modules/usage/pages/advanced_configuration_management.adoc]: the last policy generation(s)

File: src/reference/modules/usage/pages/advanced_configuration_management.adoc [hunk 3]
```diff
@@ -36,6 +36,74 @@ the regeneration of all policies. It will clear all internal caches, and force a
...
+in the following files:
+
+- `last-failure-message`: when a generation fails, failure message is store here. It is helpful for
```
Comment (ncharles) [inline src/reference/modules/usage/pages/advanced_configuration_management.adoc]: is stored

File: src/reference/modules/usage/pages/advanced_configuration_management.adoc [hunk 4]
```diff
@@ -36,6 +36,74 @@ the regeneration of all policies. It will clear all internal caches, and force a
...
+  debugging purpose (for example when message is very long), or to know what happened in case of a 
+  transcient errors. 
+- `last-updated-nodeids`: this files contains list of node updated in last generation. It is in a
```
Comment (ncharles) [inline src/reference/modules/usage/pages/advanced_configuration_management.adoc]: this file contains

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**